compilers/concrete-compiler/compiler/Makefile

```diff
@@ -362,7 +362,14 @@ $(BENCHMARK_CPU_DIR)/end_to_end_round.yaml: tests/end_to_end_fixture/end_to_end_
 $(BENCHMARK_CPU_DIR)/%.yaml: tests/end_to_end_fixture/%_gen.py
 	$(Python3_EXECUTABLE) $< > $@
 
-generate-cpu-benchmarks: $(BENCHMARK_CPU_DIR) $(BENCHMARK_CPU_DIR)/end_to_end_linalg_apply_lookup_table.yaml $(BENCHMARK_CPU_DIR)/end_to_end_apply_lookup_table.yaml $(BENCHMARK_CPU_DIR)/end_to_end_round.yaml
+
+BENCHS_CPU = \
+	$(BENCHMARK_CPU_DIR)/end_to_end_linalg_apply_lookup_table.yaml \
+	$(BENCHMARK_CPU_DIR)/end_to_end_apply_lookup_table.yaml \
+	$(BENCHMARK_CPU_DIR)/end_to_end_round.yaml \
+	$(BENCHMARK_CPU_DIR)/tests/end_to_end_fixture/end_to_end_linalg_leveled_gen.yaml
```

Comment:
this path lookup like weird, the yaml reference file should be generated on the BENCHMARK_CPU_DIR not on sub directory to be runned

Reply:
Could you provide a dashboard that show the speedup? I think it could be great to add an example from CML to the benchmarks to track those only leveled programs.

Reply:
I just force push for the fix, let's wait for benchmarks result. But I think we should still add leveled cml examples to be benched